### PR TITLE
Vetclinic: Create a database based on a diagram

### PR DESCRIPTION
**Pull request by kelomo2502**
In this milestone:

- We created a file named schema_based_on_diagram.sql
- Add indexes to foreign keys

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+create table invoice_items(id int generated always as identity primary key,
+unit_price decimal,
+quantity int,
+total_price decimal,
+invoice_id int references invoices(id),
+treatment_id int references treatments(id));
+
+create table treatments(id int generated always as identity primary key,
+type varchar(500),
+name varchar(300));
```
Comment (ShadyShawkat) [inline schema_based_on_diagram.sql:26]: - [ ] Please add the join the table between `medical_histories` and `treatments` tables as there is a many to many relation between those tables. Also, please make sure to create an index for that join table.

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+create index patient_id_index ON medical_histories(patient_id);
+creaate index medical_history_index ON invoices(medical_history_id);
```
Comment (ShadyShawkat) [inline schema_based_on_diagram.sql]: - [ ] Please correct this typo in the word `creaate`.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @kelomo2502, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smunozmo — APPROVED** [review]
Hi @kelomo2502, 👋 

## Highlights 👏 

- Excellent job making the suggested changes, your project is complete! 😎 
- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

# APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**